src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
```

Comment:
1. [Checklist] Make names descriptive, so they explain the purpose of variables and functions.
and for the next lines

Reply:
are u seriosly ? -_-

Reply:
I think it would be better to describe what you return as a final result (totalCost)

Reply:
@vserhiichuk @olegpaunov 

Try to be more clear in the comments. Describe the error more precisely, and when you don't agree with the comment please provide arguments why you should leave it as it is